### PR TITLE
Bug Fixes: Color and Download All

### PR DESCRIPTION
**Pull request by jaybythebay**
CLI was erroring on 2 issues:
* Not having a default Tableau API Version
* Not having an IF statement before a colored print

### CODE REVIEW

**Comment by jaybythebay** [comment]
@JustinGrilli  a couple of minor bugs

**Review by JustinGrilli — CHANGES_REQUESTED** [review]
Can you revert the default api_version change, and pull the change I just pushed?
If that change I pushed doesn't fix this, we can troubleshoot from there :)

File: tableau_utilities/scripts/cli.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
     '-sn', '--site_name',
     help='Site name. i.e. <site> in https://<server_address>.online.tableau.com/#/site/<site>'
 )
-group_server_auth.add_argument('--api_version', help='Tableau API version')
+group_server_auth.add_argument('--api_version', default='3.18', help='Tableau API version')
```
[the remark this answers is not in the document]
Comment (jaybythebay) [reply tableau_utilities/scripts/cli.py]: @JustinGrilli  your fix did the trick.

Here are 2 other small bug fixes if you want to merge this in.

**Review by JustinGrilli — APPROVED** [review]
Looks good! Thanks Jay!

**Comment by jaybythebay** [comment]
Thanks to you as well!

**Pull request merged**